src/database/pdf_convert.py

```diff
@@ -0,0 +1,106 @@
+import os
+import requests # pip install requests
+
+# The authentication key (API Key).
+# Get your own by registering at https://app.pdf.co
+API_KEY = "ymao@olin.edu_79f0f0b7a25c01d3b49b57614386df889907"
```

Comment:
API keys should not be stored explicitly is a file on GitHub

Reply:
Should they get their own API key? If yes, include this information in the `README.md`. I would change this line to the python version of a system variable reference. This is how it is done in the build.gradle folder when calling the Google Maps API: `"$System.env.GOOGLE_MAPS_API_KEY"`